content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,134 @@
+
+#. Select the :guilabel:`Destination Folder`.
+
+   .. warning::
+      Odoo's Windows virtual IoT software shouldn't be installed inside any of the Window's User's
+      directories. Doing so won't allow for Nginx to initialize.
+
+   Choosing ``C:\odoo`` as the install location will allow for the Nginx server to start. Click
+   :guilabel:`Install`. If the folder doesn't exist, then create it. Otherwise the installation
+   files will be spread throughout the hard drive.
+
+#. Allow the installation to occur. This may take a few minutes. Click :guilabel:`Next` to continue.
+
+#. Ensure that the :guilabel:`Start Odoo` box is checked and click :guilabel:`Finish`.
+
+#. After installation, the Odoo server will run and automatically open ``localhost:8069`` on your
+   web browser. The webpage should display the :abbr:`IoT (Internet of Things)` Box configuration
+   page.
+
+   .. seealso::
+      A restart of the Windows IoT program may be necessary should the web browser not display
+      anything. :ref:`restart_windows_iot`
+
+#. Connect :abbr:`IoT (Internet of Things)` devices to the Windows computer.
+
+#. Windows should automatically detect the device because the driver is pre-installed on
+   the computer. If not, search for and install the Windows driver for the device.
+
+#. Refresh the :abbr:`IoT (Internet of Things)`  Box Configuration Page and verify the device is
+   seen on the configuration page. If not, reload the handlers through the configuration page.
+
+#. Connect Windows :abbr:`IoT (Internet of Things)` to a database using existing instructions
+   (manually using the Token).
+
+   .. seealso::
+      :doc:`connect`
+
+#. Use the devices connected to :abbr:`IoT (Internet of Things)` as you normally would.
+
+
+Troubleshooting
+===============
+
+.. _restart_windows_iot:
+
+Restart Windows IOT box
+-----------------------
+
+In some instances a manual restart of the physical :abbr:`IoT (Internet of Things)` box can resolve
+the issue of an :abbr:`IoT (Internet of Things)` box not showing up on the database. For the Windows
+virtual :abbr:`IoT (Internet of Things)` box a manual restart of the Odoo server can resolve
+database connection issues.
+
+To restart the virtual Windows IoT server:
+
+#. Type "Services" into the :guilabel:`Search Bar`
+#. Select the :menuselection:`Services` App and scroll down to the :guilabel:`Odoo` Service.
+#. Right click on :guilabel:`Odoo` and select :guilabel:`Start` or :guilabel:`Restart`. This action
+   will manually restart the Odoo IoT server.
+
+Firewalls
+---------
+
+The Windows virtual :abbr:`IoT (Internet of Things)` box software may not be reachable to the
+:abbr:`LAN (Local Area Network)` due to a firewall preventing the connection. Consult your local IT
+support to make exceptions (network discovery) in the :abbr:`OS (Operating System)` or firewall
+program. Windows has their own firewall as well as other virus protection do too.
```

Comment:
From a support point of view, I am a bit afraid that this part will be overlooked by customers as it is "too technical".
Don't you think it would be better to use an actual example ? Like:

"I can see the IoT box homepage from the Windows virtual computer, but can not access it from other computers/mobiles/tablet in the same network"

speaking about LAN will confuse most of the customers and we (the support) will likely lose time redirecting then to this part of the documentation. It might also be a good occasion to add a .seealso to
https://www.odoo.com/documentation/16.0/applications/productivity/iot/config/troubleshooting.html#my-iot-box-is-connected-to-my-database-but-cannot-be-reached

What do you think ?

Reply:
I've added this example into the doc. Thanks for suggesting @lse-odoo